org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/completion/LSContentAssistProcessor.java

```diff
@@ -137,6 +137,11 @@ public ICompletionProposal[] computeCompletionProposals(ITextViewer viewer, int
 								if (isIncomplete) {
 									anyIncomplete.set(true);
 								}
+							}).exceptionally(t -> {
+								if (!CancellationUtil.isRequestCancelledException(t)) {
```

Comment:
do we need now lines 152-154?

Reply:
No thos lines are needed any more indeed. Corrected as well